numba/tests/test_target_extension.py

```diff
@@ -41,6 +42,7 @@
 from numba.core.compiler import CompilerBase, DefaultPassBuilder
 from numba.core.compiler_machinery import FunctionPass, register_pass
 from numba.core.typed_passes import PreLowerStripPhis
+from numba.tests.test_extending import mydummy_type, MyDummyType
```

Comment:
Same comment as above RE used of `self.make_dummy_type()` within the test.

Reply:
Changed.